calc/calculator.py

```diff
@@ -0,0 +1,31 @@
+import sys
+
+
+class Calculator:
+    @staticmethod
+    def add(x, y): return x + y
+
+    @staticmethod
+    def sub(x, y): return x - y
+
+    @staticmethod
+    def mul(x, y): return x * y
+
+    @staticmethod
+    def div(x, y): return x / y
+
+
+x= float(sys.argv[1])
```

Comment:
1) Use another python file for working calculator implementation
2) Use your calculator class methods in your application
3) Use right branch for pull request
4) Update pull request name